src/modules/75-cf/strings/strings.en.yaml

```
@@ -622,7 +622,14 @@ onboarding:
   secret: SDK Secret
   clientKey: Client SDK Key
   setUpYourCode: Set up your code
-  listenToEvent: We’re listening for the event of your feature flag.
+  listenToEvent: We're listening for the change to your flag status...<br><br>Be patient, it may take up to 3-5 minutes
+  validatingYourFlag: Validating your Flag
+  validatingFlagInfo: 'Enable your Flag to verify that the event we received has the same value. Learn more about <a target="_blank" href="https://docs.harness.io/article/7ikyqtmjce-communication-sdks-harness-feature-flags">Events and Evaluation</a>'
+  eventWeReceived: Event we received
+  createdAndValidated: Congrats! You have created and validated your first Flag! <br><br>Now you can define which Targets and Target groups this flag should be enabled for, and then see how that affects your application.
+  errorReceivingEvent: We couldn't receive a Flag status from your application, check the <a target="_blank" href="https://docs.harness.io/category/eyzngtupao-sdk-overview"> SDK reference guide </a>to ensure you've set up the SDK correctly, then toggle the Flag to try again.
+  furtherAssistance: '<br>If you are still having issues, '
+  toggleError: We couldn’t receive events, check possible reasons below and toggle the Flag to try again
   toggleLabel: Enable or disable your flag by turning the flag toggle on or off, then verify the event we receive has the same value as your application.
```

I'm aware this isn't a part of PR
```suggestion
  toggleLabel: Enable or disable your Flag by turning the Flag toggle on or off, then verify the event we receive has the same value as your application.
```

---

Have addressed thank you